server/models/user.model.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const userSchema = new Schema({
+  email: {
+    type: String,
+    required: true,
+    unique: true
+  },
+  name: {
+    type: String,
+    required: true
+  }
+}, {
```

Comment:
shouldn't user schema also include passwords?

Reply:
This PR is still work-in-progress and not ready for final review